mitodl/mit-open
# Multiple filter options for learningresources and contenfiles API rest endpoints

### What are the relevant tickets?
Closes #293 

### Description (What does it do?)
Allows both the learning_resources and contentfiles API endpoints to filter by multiple values for the same attribute, via two different query parameter formats, similar to the search API endpoint.  Examples:

http://localhost:8063/api/v1/learning_resources/?platform=ocw,xpro
http://localhost:8063/api/v1/learning_resources/?platform=ocw&platform=xpro
http://localhost:8063/api/v1/learning_resources/?platform=ocw


Also gets rid of some contentfiles filters that are not present in the search api (run_readable_id, resource_readable_id)


### How can this be tested?
Try variations of the above urls for different filterable fields, on both the contentfiles and learning_resources API endpoints.  They should return expected results that match the same filters for the search API endpoints.

### Additional Context
